### ISSUE TITLE
import/namespace is wrong when extending interfaces

### ISSUE THREAD

**Issue by jeffrson** [benmosher/eslint-plugin-import#2040]
Hi,
since we started using typescript-eslint 4, we are facing an error, where something like
`interface ITest1_0 extends lib0.ITest1`
triggers import/namespace. When declaring variables it works, however.

This will happen especially when importing some library via "types" in package.json.

BTW, there is a warning about the library not exporting anything in this case, unless its index.ts exports something (does not need to be related to those interfaces).

I hope the repo at https://github.com/jeffrson/repro_interface_import will show the cases.

**Comment by jeffrson** [comment]
Only after I wrote this report I learned about https://github.com/alexgorbatchev/eslint-import-resolver-typescript

When I install it and enable it through
```
settings: {
        'import/resolver': {
            'typescript': {}
        }
    }
```

the error is gone. (Actually, my repro is causing two "import/no-duplicates" messages, but that's obvious because it's indeed trying to load the same module via package.json *and* as a folder from the workspace.)

Are we required to use this additional resolver? Or should it work with this plugin alone?

Nevertheless, the behaviour seems a bit strange.

**Comment by ljharb** [comment]
Yes, if you're using typescript you absolutely need the typescript resolver. The only thing that works by default is node, since that's the standard/default.

**Comment by JounQin** [comment]
close in favor of https://github.com/import-js/eslint-import-resolver-typescript
